Classes/Domain/Factory/FrontendContainerFactory.php

```diff
@@ -34,16 +34,16 @@ public function buildContainer(ContentObjectRenderer $cObj, Context $context, ?i
         if ($uid === null) {
             $record = $cObj->data;
         } else {
-            $records = $cObj->getRecords('tt_content', ['where' => 'uid=' . $uid]);
+            $records = $cObj->getRecords('tt_content', ['uidInList' => $uid, 'pidInList' => 0]);
```

Comment:
we could use pidInList = this

Reply:
i think no, because record is not on current page